pkg/notifications/email.go

```diff
@@ -43,6 +43,7 @@ func newEmailNotifier(c *cobra.Command, acceptedLogLevels []log.Level) t.Notifie
 	port, _ := flags.GetInt("notification-email-server-port")
 	tlsSkipVerify, _ := flags.GetBool("notification-email-server-tls-skip-verify")
 	delay, _ := flags.GetInt("notification-email-delay")
+	subjecttag, _ := flags.GetString("notification-email-subjecttag")
```

Comment:
The flag also needs to be added to internal/flags/flags.go, I'd actually prefer if it had a overridable default value, like the previous "Watchtower Updates" and replaced the subject altogether.

Reply:
Ok. I will think about it.